File: C/Colleen/src/Colleen.c
#include<stdio.h>
/*
    com1
*/
void print(char *a)
{
    printf(a,10,10,10,10,10,10,10,10,10,10,10,10,10,34,a,34,10,10,10);
}
int main(void)
{
    /*
        com2
    */
    char *a="#include<stdio.h>%c/*%c    com1%c*/%cvoid print(char *a)%c{%c    printf(a,10,10,10,10,10,10,10,10,10,10,10,10,10,34,a,34,10,10,10);%c}%cint main(void)%c{%c    /*%c        com2%c    */%c    char *a=%c%s%c;%c    print(a);%c}%c";
    print(a);
}

File: Python/Colleen/Colleen.py
#!/usr/bin/env python3

import sys

# com1

def colleen(a: str) -> None:
    print(a % repr(a))

def main() -> int:
    # com2
    a = f"#!/usr/bin/env python3\n\nimport sys\n\n# com1\n\ndef colleen(a: str) -> None:\n    print(a %% repr(a))\n\ndef main() -> int:\n    # com2\n    a = f%s\n    colleen(a)\n    return 0\n\nif __name__ == '__main__':\n    sys.exit(main())"
    colleen(a)
    return 0

if __name__ == '__main__':
    sys.exit(main())
